data/stages.lua

```diff
@@ -0,0 +1,33 @@
+-- set to true to enable experience stages
+stagesEnabled = false
```

Comment:
I guess this is already defined in config.lua and should still be

Reply:
really, i forgot it!